petab/problem.py

```diff
@@ -104,6 +104,31 @@ def __setattr__(self, name, value):
         else:
             super().__setattr__(name, value)
 
+    def __str__(self):
+        model = f"with model ({self.model})" if self.model else "without model"
+        conditions = f"{self.condition_df.shape[0]} conditions" \
+            if self.condition_df is not None else "without conditions table"
+
+        observables = f"{self.observable_df.shape[0]} observables" \
+            if self.observable_df is not None else "without observables table"
+
+        measurements = f"{self.measurement_df.shape[0]} measurements" \
+            if self.measurement_df is not None \
+            else "without measurements table"
+
+        if self.parameter_df is not None:
+            num_estimated_parameters = sum(self.parameter_df[ESTIMATE] == 1) \
+                if ESTIMATE in self.parameter_df \
```

Comment:
`ESTIMATE` is a required column

Reply:
True. Not sure how much we want to assume/require correctness here. Should it just fail if this is missing? Should we validate in __str__? 
Might be more convenient to allow for some slack, if this is used during assembling a Problem. Similar to libsbml where one is allowed to create invalid models.